azure-stack/includes/hci-download-vhdx.md

```diff
@@ -0,0 +1,32 @@
+---
+author: alkohli
+ms.author: alkohli
+ms.service: azure-stack
+ms.subservice: azure-stack-hci
+ms.topic: include
+ms.date: 05/25/2023
+---
+
+
+## Download the VHDX file
+
+SDN uses a VHDX file containing either the Azure Stack HCI or Windows Server operating system (OS) as a source for creating the SDN virtual machines (VMs). The version of the OS in your VHDX must match the version used by the Azure Stack HCI Hyper-V hosts. This VHDX file is used by all SDN infrastructure components.
+
+You can download the Azure Stack HCI VHDX file from the [Azure Stack HCI software download site](https://azure.microsoft.com/contact/azure-stack-hci/). Complete the download form and select **Submit** to display the **Azure Stack HCI software download** page. To download the VHDX file, under **Azure Stack HCI**, select **English – VHDX** from the **Choose language** dropdown menu, and then select **Download Azure Stack HCI**.
+
+:::image type="content" source="./media/sdn/download-vhdx.png" alt-text="Screenshot of the Azure Stack HCI software download page showing the option to download the VHDX file." lightbox="./media/sdn/download-vhdx.png":::
+
```

Comment:
Somewhere you should say that the above instructions are for English. And then say for non-English version.......

Reply:
Done.